itests/hive-jmh/src/main/resources/log4j2.properties

```diff
@@ -14,7 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-status = FATAL
```

Comment:
Although I've not checked how Log4j 2 works, I am feeling that the original intention to see only FATAL will be gone

Reply:
Thanks for the review @okumin. Very valid question, when I was working on the PR, i also though about it.
Based on the comments https://github.com/apache/logging-log4j2/issues/2794#issuecomment-2267564572 
> Status Logger is intended for troubleshooting Log4j itself and its components. I strongly advise you to avoid configuring it unless you are troubleshooting a Log4j issue.

Because of that I went ahead and removed. I don't have any strong opinion on this, If you think its better to keep, then I can update the PR.

Reply:
Thanks. I agree with your decision 👍